src/Features/CSharp/Portable/Completion/KeywordRecommenders/SwitchKeywordRecommender.cs

```diff
@@ -16,7 +16,8 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
         {
             return
                 context.IsStatementContext ||
-                context.IsGlobalStatementContext;
+                context.IsGlobalStatementContext ||
+                context.IsIsOrAsContext; // ie. following an expression
```

Comment:
this seems super bizarre.  If you really want to reuse this property, then call it IsIsOrAsOrSwitchContext